pkg/platform/model/projects.go

```diff
@@ -33,6 +33,9 @@ var (
 
 	// FailProjectNameConflict is a failure due to a project name conflict
 	FailProjectNameConflict = failures.Type("model.fail.projectconflict")
+
+	// FailProjectNotFound is a fialure due to a project not being found
+	FailProjectNotFound = failures.Type("model.fail.projectnotfound")
```

Comment:
This failure was added to satisfy the last AC.
> If project doesn't exist remotely yet then it should print a simple notice (along with the local information):
"Please run state push to synchronize this project with the ActiveState Platform."

I attempted to convert this function to returning an error however it would lead to a very large changeset